src/redis/t_stream.c

```diff
@@ -345,7 +345,12 @@ int64_t streamTrim(stream *s, streamAddTrimArgs *args) {
 
         if (remove_node) {
             lpFree(lp);
-            raxRemove(s->rax_tree,ri.key,ri.key_len,NULL);
+            int res = raxRemove(s->rax_tree,ri.key,ri.key_len,NULL);
+            if(res == 0) {
```

Comment:
This should not happen but I added this for sanity.

Reply:
why it should not happen? how do you know?